controllers/articles.controller.js

```diff
@@ -20,3 +21,27 @@ exports.getArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+
+// exports.updateVotes = (req, res, next) => {
+//   const newVotes = req.body;
+//   const article_id = req.params.article_id;
+//   changeVotes(newVotes, article_id)
+//     .then((article) => {
+//       return res.status(200).send(article);
+//     })
+//     .catch(next);
+// };
+
+exports.updateVotes = (req, res, next) => {
+  const newVotes = req.body;
+  const article_id = req.params.article_id;
+  const promises = [changeVotes(newVotes, article_id)];
+  if (article_id) {
+    promises.push(checkArticleExists(article_id));
+  }
```

Comment:
There should always be an article id! Checking that it's there is not necessary. 

It's also not necessary to check that the article exists in this case - the article_id is a foreign key so if it doesn't exist psql will throw an error itself! But you've not technically done anything wrong it could just be handled a different way!